modules/covid_vaccine/app/services/covid_vaccine/v0/enrollment_service.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'net/sftp'
+require 'covid_vaccine/v0/expanded_registration_submission_csv_generator'
+
+module CovidVaccine
+  module V0
+    class EnrollmentService
+      def initialize(records)
+        @records = records
+        @io = ExpandedRegistrationSubmissionCSVGenerator.new(@records).io
+      end
+
+      attr_reader :records, :io
+
+      def send_enrollment_file(file_name_prefix: '')
+        file_name = file_name_prefix + generated_file_name
+
+        Net::SFTP.start(sftp_host, sftp_username, password: sftp_password) do |sftp|
+          sftp.upload!(@io, remote_file_path(file_name), name: file_name, progress: EnrollmentHandler.new)
+        end
+      end
+
+      private
+
+      def remote_file_path(name)
+        "/#{name}"
+      end
+
+      def generated_file_name
```

Comment:
Can you make it so DHS_load is a default param and if pass in something else like TEST_ it doesn't even include 'DHS_load', that way if their pattern matching is not great, a TEST file won't have that string anywhere in it.

Reply:
Maybe that's over-thinking it

Reply:
I think he said if it's prefixed with anything other than DHS_load it wont pick up automatically

Reply:
nevermind, but we can make sure whatever rake tasks we put in place are safe w/r/t filenames